Malformed XML

It seems that `./source/docs/01-introduction.xml` is malformed in the `./release/v5.0` branch, causing `ant` to fail.
```bash
Buildfile: /Users/laurent/libs/mei_official/build.xml

init:
     [echo] xerces available: true
     [echo] saxon available: true
     [echo] prince available: false
     [echo] initialized

init-mei-classpath:
     [echo] mei.classpath set

canonicalize-source:
     [echo] canonicalizing source...
     [java] Error on line 305 column 21 of 01-introduction.xml:
     [java]   SXXP0003   Error reported by XML parser: The element type "p" must be terminated by the
     [java]   matching end-tag "</p>".: The element type "p" must be terminated by the matching end-tag "</p>".
     [java] Error on line 113 column 79 of mei-source.xml:
     [java]   SXXP0003   Error reported by XML parser: Error attempting to parse XML file
     [java]   (href='docs/01-introduction.xml').: Error attempting to parse XML file
     [java]   (href='docs/01-introduction.xml').
     [java] org.xml.sax.SAXParseException; systemId: file:/Users/laurent/libs/mei_official/source/mei-source.xml; lineNumber: 113; columnNumber: 79; Error attempting to parse XML file (href='docs/01-introduction.xml').

BUILD FAILED
/Users/laurent/libs/mei_official/build.xml:204: Java returned: 2
```

--- a/source/docs/01-introduction.xml
+++ b/source/docs/01-introduction.xml
@@ -301,16 +301,17 @@
                     our Sample Encodings and Encoding Tools repositories. Sample Encodings have been updated
                     to MEI 5.0, and several problems with encodings from older releases have been fixed. In the
                     Encoding Tools, several bugs were fixed with older upgrade XSLT scripts, and a new XSLT for
-                    upgrading MEI 4 to MEI 5 was added.
-                  </div>
-                  <p>To see all of the changes made for this revision, please visit our Git repositories:
+                    upgrading MEI 4 to MEI 5 was added.</p>
+                    <p>To see all of the changes made for this revision, please visit our Git repositories:
                     <list rend="simple">
                       <item><ref target="https://github.com/music-encoding/music-encoding">https://github.com/music-encoding/music-encoding</ref></item>
                       <item><ref target="https://github.com/music-encoding/sample-encodings">https://github.com/music-encoding/sample-encodings</ref></item>
                       <item><ref target="https://github.com/music-encoding/encoding-tools">https://github.com/music-encoding/encoding-tools</ref></item>
                     </list>
-                  <p>The editors wish to thank everyone who participated in this process. Of course, 
-                  errors and omissions are the sole responsibility of the editors.</p>
+                    </p>
+                    <p>The editors wish to thank everyone who participated in this process. Of course, 
+                    errors and omissions are the sole responsibility of the editors.</p>
+                  </div>
                </div>
             </div>
             <div xml:id="designPrinciples" type="div2">

